test/integration/README.md

```diff
@@ -4,23 +4,24 @@
   * See this directory's package.json
 * TAP
   * In the scratch-www repo's package.json
-* Currently, we also require that you download [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/), but this will be replaced by Saucelabs imminently
+* Currently, we also require that you download [Chromedriver](https://sites.google.com/a/chromium.org/chromedriver/), but this will be replaced by Saucelabs eventually
```

Comment:
Maybe we should just remove references to Sauce altogether, and we can just add it back when we have something? When we do get to it, we may not even end up using that service (although we probably will).

Reply:
Sure, will do.